src/DotNetLightning.Core/Transactions/Transactions.fs

```diff
@@ -297,6 +298,13 @@ module Transactions =
         [<Literal>]
         let OFFERED_HTLC_SCRIPT_WEIGHT = 133uy
 
+    let CreateTransactionBuilder (network: Network) =
+        let txb = network.CreateTransactionBuilder()
+        txb.ShuffleOutputs <- false
+        txb.ShuffleInputs <- false
+        txb
+
+    let UINT32_MAX = 0xffffffffu
```

Comment:
1. the name is confusing, because `UInt32.MaxValue` is `0xffffffff`, what about `UINT32_MAX_MINUS_1`
2. Using `[<Literal>]` will improve the performance a bit.

Reply:
Oh never mind I misread.